glslc/src/main.cc

```diff
@@ -135,9 +135,14 @@ An input file of - represents standard input.
                     generation.
   -S                Only run preprocess and compilation steps.
   --target-env=<environment>
-                    Set the target shader environment, and the semantics
-                    of warnings and errors. Valid values are 'opengl',
-                    'opengl_compat' and 'vulkan'. The default value is 'vulkan'.
+                    Set the target client environment, and the semantics
+                    of warnings and errors.  An optional suffix can specify
+                    the client version.  Values are:
+                        vulkan1.0       # The default
+                        vulkan1.1
+                        vulkan          # Same as vulkan1.0
```

Comment:
A question  about the `vulkan` parameter suddenly comes to me.

What's our guarantee of this parameter? Are we gonna change what it aliases to when, say, Vulkan 1.2 or 1.N releases?

If not, that effectively makes it just a way of saying `vulkan1.0`, which would be handy for now but in the future developers will mostly want vulkan 1.N. So it will likely be less useful.

If we would like to change it when newer version of Vulkan come out, then how we decide which version to alias to since we are not alias to the newest version here?

Also note that when we have a `python` or other binary or library without version number in the system, they alias to the newest ones. If  you want to the older ones, you need to specify `python2.6` or something. I'm not sure whether that's particularly related here, but seems it's a convention?

Reply:
Okay, I think maybe make `vulkan` alias to the previous major release is a good balance between new functionality and driver stability?

Reply:
JohnK and I independently came to the same conclusion that at least for now defaulting to Vulkan 1.0 is a sane default.  It also happens to retain backward compatibility.
It is a judgement call though.

Reply:
That's fair.